server/api/orders.js

```diff
@@ -1,39 +1,55 @@
 const router = require('express').Router();
-const { Order, LineItem } = require('../db/models');
-const { requireAdmin, requireLogin, requireUserOrAdmin } = require('./util');
+const {Order, LineItem, Product, Image} = require('../db/models');
+const {requireAdmin, requireLogin, requireUserOrAdmin} = require('./util');
 
 //Get single order
 
-router.get(
-  '/:id',
-  requireLogin,
-  requireUserOrAdmin,
-  (req, res, next) => {
-    return Order.findById(req.params.id)
-      .then(order => res.json(order))
-      .catch(next);
-  }
-);
+router.get('/', requireLogin, requireUserOrAdmin, async (req, res, next) => {
+	try {
+		const orders = await Order.findAll();
+		res.json(orders);
+	} catch (error) {
+		next(error);
+	}
+});
+
+//Get single order
+
+router.get('/:id', requireLogin, requireUserOrAdmin, (req, res, next) => {
```

Comment:
requireUserOrAdmin uses req.params.id to check user id. But here, it is orderid. So this makes a problem. I would suggest something like /api/orders/userId/orderId to show single order.  /:id/:orderId

Users route. api/users/:id/orders to serve all orders a list. Use req.user.id to find all orders belong to that user.